source/cloud/cloud-administration/cloud-changelog.md

```diff
@@ -2,6 +2,43 @@
 
 This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.
 
+## Release 2021-03-10
+
+### Highlights
+
+#### Custom Statuses (Beta)
```

Comment:
Remove (Beta) label

Reply:
@esethna Is there a specific reason why it's not beta since it doesn't have mobile support yet? Documentation here https://handbook.mattermost.com/operations/research-and-development/product/product-management-team-handbook#can-we-take-this-feature-out-of-beta. Just curious about the "Why?" so that I understand.

Reply:
I think we need to review each feature on a case by case basis. Having a shortage of mobile engineers we'll struggle to keep up with webapp and we can't hold all features in beta. We also have no way to identify in the UI that this feature is "beta" if so.

Given mobile is coming soon I'm not concerned about releasing as "GA" with a note that mobile is coming soon in the blog

Reply:
Thank you!